File: engine/model/src/main/scala/tools/packet/alliance/GetAlliancePlayerInfo.scala
package tools.packet.alliance

import net.opcodes.SendOpcode
import tools.packet.PacketInput

class GetAlliancePlayerInfo( private var _allianceId: Int,  private var _playerId: Int) extends PacketInput {
     def allianceId: Int = _allianceId
     def playerId: Int = _playerId

  override def opcode(): SendOpcode = SendOpcode.ALLIANCE_OPERATION
}

File: engine/engine-model/src/main/java/tools/packet/alliance/GetAlliancePlayerInfo.java
package tools.packet.alliance;

import net.opcodes.SendOpcode;
import tools.packet.PacketInput;

public record GetAlliancePlayerInfo(Integer allianceId, Integer playerId) implements PacketInput {
   @Override
   public SendOpcode opcode() {
      return SendOpcode.ALLIANCE_OPERATION;
   }
}
